.ci.yaml

```diff
@@ -4284,8 +4283,19 @@ targets:
         ["devicelab", "ios", "mac"]
       task_name: route_test_ios
 
-  - name: Mac_ios flavors_test_ios
+  - name: Mac_x64_ios flavors_test_ios
```

Comment:
I know you're basing this off go/flutter-mac-test-architectures and I have `flavors_test_ios` as "Both" but looking at it again I don't know why that would be needed.  Could you instead implement this as an "Any"?  Sorry about that.
And `flavors_test_macos` didn't exist when I wrote that doc, same thing, could be "Any".

Reply:
Sounds good, reverted it back to any.